KomaMRIFiles/src/Sequence/Pulseq.jl

```diff
@@ -569,8 +575,14 @@ Reads the ADC. It is used internally by [`get_block`](@ref).
 - `adc`: (`1x1 ::Vector{ADC}`) ADC struct
 """
 function read_ADC(adcLibrary, i)
+
+    if isempty(adcLibrary) || i==0
+        return [ADC(0, 0)]
+    end
```

Comment:
If you are going to add this.

Reply:
Addressed in [5baa0a0](https://github.com/JuliaHealth/KomaMRI.jl/pull/321/commits/5baa0a04f4da6c922c7b6851a8400d3081af278a)